.github/workflows/release.yml

```diff
@@ -1,7 +1,8 @@
 name: Release
+
 on:
-  release:
-    types: [ "published" ]
+  push:
+    tags: 'v*.*.*'    
```

Comment:
Wouldn't just `v*` (or more specific `v\d+`) work here?

Reply:
I honestly don't remember why this PR even exists. Shall I close it?